<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEditCoffeeForm</class>
 <widget class="QWidget" name="addEditCoffeeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Добавить, редактировать</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Введите ID строки</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="text">
      <string>Редактировать строку</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>Добавить строку</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>Сохранить измененния</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Название сорта</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Степень обжарки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Молотый/в зернах</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Описание вкуса</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Объем упаковки</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_info">
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
